src/__test__/Memes.test.jsx

```diff
@@ -0,0 +1,31 @@
+import renderer from 'react-test-renderer';
```

Comment:
- [ ] Some of your tests are failing, kindly update them 
![image](https://github.com/haftamudesta/React-Capstone-Project/assets/62705947/bcb2f90f-7303-445f-9ef9-b7919b8bf8f2)

Reply:
Well resolved ✔️ 

![image](https://github.com/haftamudesta/React-Capstone-Project/assets/38088818/4aaf6f15-0afb-46cd-bd19-82d18ee933ef)